modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/routes/customer-portal/containers/ActivationKeysTable/index.jsx

```diff
@@ -82,17 +82,18 @@ const ActivationKeysTable = ({
 		statusfilterByTitle: [statusFilter, setStatusFilter],
 	} = useStatusCountNavigation(activationKeys);
 
-	const {activationKeysByStatusPaginated, paginationConfig} = usePagination(
-		activationKeys,
-		statusFilter
-	);
-
 	const [filters, setFilters] = useFilters(
 		setFilterTerm,
 		productName,
 		initialFilter
 	);
 
+	const {activationKeysByStatusPaginated, paginationConfig} = usePagination(
+		activationKeys,
+		statusFilter,
+		filters
```

Comment:
@samuelleolopes remove this, we don't use on usePagination

```suggestion
	
```

Reply:
Fixed, thank you!